examples/pytorch/simple.yaml

```diff
@@ -16,7 +16,7 @@ spec:
               imagePullPolicy: Always
               command:
                 - "python3"
-                - "/opt/mnist/src/mnist.py"
+                - "/opt/pytorch-mnist/mnist.py"
```

Comment:
This is correct in the master branch.
Please see https://github.com/kubeflow/training-operator/blob/ede74760df54985fbd650495a165c5f80af8d6d1/examples/pytorch/mnist/Dockerfile#L15 for more details.

Reply:
Sorry, my understanding is not correct since we are using the katib image here: https://github.com/kubeflow/katib/blob/21320b6d5774e4a7828a2c49e4bff75cd55e6b35/examples/v1beta1/trial-images/pytorch-mnist/Dockerfile.cpu#L17

/reopen